packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -223,7 +223,7 @@ const browsers = computed(() => {
     return undefined
   }
 
-  return [...props.gql.browsers].sort((a, b) => a.name === 'Electron' ? 1 : -1)
```

Comment:
Can we move this sort logic to frontend-shared/utils or otherwise externalize it so that it's harder for the app and launchpad to fall out of sync again? It's straightforward now, but sounds like it could get more complicated in the future with usage metrics factored in.